src/gtk/textctrl.cpp

```diff
@@ -124,6 +124,41 @@ static void wxGtkTextApplyTagsFromAttr(GtkWidget *text,
         }
     }
 
+    if( attr.HasFontUnderlinedWithEffect() )
+    {
+        GtkTextTag *tag1;
```

Comment:
This is not a great variable name. How is anybody supposed to know what does it stand for?

Reply:
@vadz,
How anybody should know what all other "tag" variable means in this function?
But I will rename it nevertheless.